### PR TITLE
Updates some styles and the first two pages

### PR DESCRIPTION
**Pull request by kamronbatman**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -4,42 +4,51 @@ title: Installation
...
 # Installation
 
-## Prerequisites
-
 === "Windows"
-    1. Download and install git from [here](https://git-scm.com/download/win)
-    2. Download and install the latest [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+    ### Prerequisites
+    1. Download and install the latest [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+    1. Download and install from [here](https://git-scm.com/download/win)
```
Comment (deccer) [inline docs/installation.md:10]: Typo? should be 2.

File: docs/installation.md [hunk 2]
```diff
@@ -4,42 +4,51 @@ title: Installation
...
+
+        !!! Tip
+            Use Git Bash as your command prompt. This can be found in the Windows Start Menu after installation.
+
+    ### Clone the Repository (Install ModernUO)
+    1. Navigate to the folder where you want to install ModernUO.
+    1. Using _Git Bash_ run:
```
Comment (deccer) [inline docs/installation.md:17]: should be 2. too, i suppose

File: docs/installation.md [hunk 3]
```diff
@@ -4,42 +4,51 @@ title: Installation
...
+        git clone https://github.com/modernuo/modernuo
+        cd modernuo
+        ```
 
 === "OSX"
-    1. Install homebrew from [here](https://brew.sh)
-    2. Run:
+    ### Prerequisites
+    1. Download and install the latest [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+    1. Using _terminal_, install [homebrew](https://brew.sh) and git:
```
Comment (deccer) [inline docs/installation.md:26]: same here

File: docs/installation.md [hunk 4]
```diff
@@ -4,42 +4,51 @@ title: Installation
...
+       ```bash
+        git clone https://github.com/modernuo/modernuo
+        cd modernuo
+       ```
 
 === "Linux"
-    1. Run:
+    ### Prerequisites
+    1. Download and install the latest [.NET Core SDK](instructions [here](https://docs.microsoft.com/en-us/dotnet/core/install/linux))
+    1. Using _bash_, install git:
```
Comment (deccer) [inline docs/installation.md:41]: aaaand here

**Review by deccer — CHANGES_REQUESTED** [review]
Besides the typos, looks ok to me

**Comment by kamronbatman** [comment]
Markdown auto numbers based on consecutive instances of 1.

1. This
1. This

**Comment by deccer** [comment]
I am not happy about that behaviour. I like consistency :) But still fine with me.

**Review by deccer — APPROVED** [review]

**Pull request merged**